lib/core/Axios.js

```diff
@@ -34,7 +34,7 @@ Axios.prototype.request = function request(config) {
     }, arguments[1]);
   }
 
-  config = utils.merge(defaults, this.defaults, { method: 'get' }, config);
+  config = utils.merge(defaults, { method: 'get' }, this.defaults, config);
```

Comment:
I'd move the default method to the `defaults` file so we don't have exceptions in other parts of the code.